### ISSUE TITLE
AttributeError: function '__gmpz_init' not found

### ISSUE THREAD

**Issue by truekonrads** [Legrandin/pycryptodome#131]
It would seem that on Windows the wheel isn't correctly built. 

Related to https://github.com/google/rekall/issues/359

(rekall) C:\Tools\rekall>ver
Microsoft Windows [Version 6.1.7601]
(rekall) C:\Tools\rekall>python --version
Python 3.6.4

```

(rekall) C:\Tools\rekall>pip install --upgrade pycryptodome
Collecting pycryptodome
  Downloading pycryptodome-3.4.11-cp36-cp36m-win_amd64.whl (7.3MB)
    100% |████████████████████████████████| 7.3MB 59kB/s
Installing collected packages: pycryptodome
  Found existing installation: pycryptodome 3.4.7
    Uninstalling pycryptodome-3.4.7:
      Successfully uninstalled pycryptodome-3.4.7
Successfully installed pycryptodome-3.4.11

(rekall) C:\Tools\rekall>rekaLL
Traceback (most recent call last):
  File "c:\tools\python36\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\tools\python36\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Tools\rekall\Scripts\rekall.exe\__main__.py", line 5, in <module>
  File "c:\tools\rekall\lib\site-packages\rekall\rekal.py", line 32, in <module>

    from rekall import args
  File "c:\tools\rekall\lib\site-packages\rekall\args.py", line 38, in <module>
    from rekall import config
  File "c:\tools\rekall\lib\site-packages\rekall\config.py", line 45, in <module
>
    from rekall_lib import yaml_utils
  File "c:\tools\rekall\lib\site-packages\rekall_lib\__init__.py", line 1, in <m
odule>
    from rekall_lib import crypto
  File "c:\tools\rekall\lib\site-packages\rekall_lib\crypto.py", line 13, in <mo
dule>
    from Crypto.PublicKey import RSA
  File "c:\tools\rekall\lib\site-packages\Crypto\PublicKey\RSA.py", line 42, in
<module>
    from Crypto.Math.Numbers import Integer
  File "c:\tools\rekall\lib\site-packages\Crypto\Math\Numbers.py", line 42, in <
module>
    from Crypto.Math._Numbers_gmp import Integer
  File "c:\tools\rekall\lib\site-packages\Crypto\Math\_Numbers_gmp.py", line 128
, in <module>
    _gmp.mpz_init = lib.__gmpz_init
  File "c:\tools\python36\Lib\ctypes\__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "c:\tools\python36\Lib\ctypes\__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: function '__gmpz_init' not found
```

**Comment by Legrandin** [comment]
Do you have a DLL called `gmp.dll` somewhere in your system?
It is loaded and then the code breaks because it is not actually a valid GMP library.

**Comment by truekonrads** [comment]
That's correct, I have some from an OpenSSL distribution (C:\xxxx\OpenSSL-Win64\bin\gmp.dll). It does a long search at a variety of places before it finds it.  What should be the fix?

**Comment by Legrandin** [comment]
Catching `AttributeError` in `Crypto.Math.Numbers` is the long term solution; that's simple but it also requires a new release and I won't manage in the coming days I think.

As a workaround, maybe removing that directory from `PATH` is an option?

**Comment by Legrandin** [comment]
And of course pinning `pycryptodome` to 3.4.7 might be easy too.

**Comment by Legrandin** [comment]
Fixed with cf1b0387b18ce5887a99d978d9b43abda89e43e0
